actions/platform/integrations/add_integration.py

```diff
@@ -35,7 +35,7 @@
     "integration_setup_kind",
     [
         FuzzySlotMatchOption("cloud"),
-        FuzzySlotMatchOption("red_hat", ["red hat", "redhat"]),
+        FuzzySlotMatchOption("red_hat", ["red hat", "redhat", "red-hat", "red hat"]),
```

Comment:
This looks like a duplicate. Perhaps it was supposed to be "rhel"

Reply:
copy/paste error- rhel would have made sense, yes